src/e2c/run_script.py

```diff
@@ -4,6 +4,12 @@
 from colorama import init as _init_colorama
 
 from e2c.db_connector import DBConnector
+from e2c.pdf_tools import (
+    _check_new_color,
+    _compare_color,
+    _extract_annot,
+    _open_file,
```

Comment:
If a function is named with `_` as prefix, they should not be imported and used, vice versa.

Reply:
The `run_script.py` file can be quite long if we move all underscored functions inside.

Reply:
You can merge as-is. But please add a new issue to indicate this needs refactoring.